crates/auth/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "spacetimedb-auth"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+
+[dependencies]
+spacetimedb-lib.workspace = true
+
+anyhow.workspace = true
+serde.workspace = true
+serde_with.workspace = true
+jsonwebtoken.workspace = true
```

Comment:
at first glance this seems unrelated to the `update` rework.. was there a circular dependency or something?

Reply:
CLI depended on this component of core, so I split it out to a separate crate. This was in the [`Remove spacetimedb-core as a dep of cli`](https://github.com/clockworklabs/SpacetimeDB/pull/2126/commits/35e54883eb11abdf19b529faec45b13b2f1c1eb5) commit.

Reply:
Was there some reason the CLI couldn't (continue to) depend on core? PR diffs get complicated quickly by having lots of changes/cleanups/refactorings that aren't related to the core change.. If you think reviewers will have an easier time reviewing commit-by-commit, please put that in the PR description because it's not necessarily the way that PRs are reviewed by default.